developers/ide/examples/vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
https://stackoverflow.com/questions/244777/can-comments-be-used-in-json

Maybe we should move this to another location.

Reply:
Was automatically added by VS - really no need for it however...